README.md

```diff
@@ -1 +1,36 @@
-# terraform-cloudtrail-bucket
+# terraform-aws-route53-endpoint
```

Comment:
I think your copy and paste made a mistake 

```suggestion
# terraform-cloudtrail-bucket
```

Reply:
fixed